src/main/java/net/dv8tion/jda/internal/entities/InviteImpl.java

```diff
@@ -360,6 +361,20 @@ public String getIconUrl()
                     : "https://cdn.discordapp.com/icons/" + this.id + "/" + this.iconId + ".png";
         }
 
+        /**
+         * Returns an {@link ImageProxy} for this guild's icon
+         *
+         * @return Possibly-null {@link ImageProxy} of this guild's icon
+         *
+         * @see    #getIconUrl()
+         */
+        @Nullable
+        public ImageProxy getIcon()
```

Comment:
Also, why is an implementation receiving documentation and there is no override? 
`@Override` should be added, and the documentation should be removed.